debian/rules

```diff
@@ -102,7 +102,7 @@ endif
 	    -DBUILD_CONFIG=mysql_release \
 	    -DCONC_DEFAULT_CHARSET=utf8mb4 \
 	    -DPLUGIN_AWS_KEY_MANAGEMENT=NO \
-			-DPLUGIN_COLUMNSTORE=NO \
+		-DPLUGIN_COLUMNSTORE=NO \
```

Comment:
This still doesn't look right. -DPLUGIN_COLUMNSTORE is still 4 characters off. Enable whitespace characters in your editor so you can see how the other lines are formed.

Reply:
Nice spot, thanks! Correct tab/space is `<tab><space><space><space><space>` and it's corrected.